gc/base/Configuration.cpp

```diff
@@ -450,6 +450,9 @@ MM_Configuration::initializeGCThreadCount(MM_EnvironmentBase* env)
 	if (!extensions->gcThreadCountSpecified) {
 		extensions->gcThreadCount = defaultGCThreadCount(env);
 	}
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	_delegate.checkPointGCThreadCountVerifyAndAdjust(env);
```

Comment:
Must provide an empty implementation of checkPointGCThreadCountVerifyAndAdjust in OMR Example VM, otherwise standalone OMR will not compile. 

Look at getMaxGCThreadCount() as an example.

Reply:
added :)